.github/CODEOWNERS

```diff
@@ -202,6 +202,8 @@ go_deps.bzl               @dfinity/idx
 /rs/rust_canisters/stable_memory_integrity              @dfinity/execution
 /rs/rust_canisters/canister_creator                     @dfinity/execution
 /rs/rust_canisters/xnet_test/                           @dfinity/ic-message-routing-owners
+/rs/rust_canisters/downstream_calls_test/		@dfinity/ic-message-routing-owners
+/rs/rust_canisters/random_traffic_test/			@dfinity/ic-message-routing-owners
```

Comment:
You also have Tab characters instead of spaces here. You can probably configure your IDE to produce spaces, not Tabs.

Reply:
What's the problem with tabs?

Reply:
They are configured to be of different size in different places. So unless you're using them everywhere (and decide on a fixed size for everyone), in some IDEs (and in GitHub diffs) they result in apparently random alignment.

If this was Python code you'd likely get an error from mixing spaces and tabs. Or, if not, you'd be likely to get incorrect behavior, depending on whether the interpreter and your IDE agreed on the width of a Tab.

Reply:
It looks fine here. I use tab all the time. But ok, here you go it's now different white space.

Reply:
![Screenshot from 2024-09-16 15-35-43](https://github.com/user-attachments/assets/25065d57-6029-4362-8de1-f680c8320dbd)